projects/packages/my-jetpack/_inc/components/product-card/index.jsx

```diff
@@ -104,7 +104,7 @@ const ProductCard = props => {
 	const canDeactivate = ( isActive || isError ) && admin;
 
 	const containerClassName = classNames( styles.container, {
-		[ styles.absent ]: isAbsent,
+		[ styles.plugin_absent ]: isAbsent,
```

Comment:
I prefer to keep the styles related only as `absent`, the same as the constant key. The only update needed was line 19.

Reply:
I merged :-(.
Anyway, the statuses defined for the Product are [defined in the backend](https://github.com/Automattic/jetpack/blob/9beef49900ad0ff6bc5ead7b933c2e6491fbe22f/projects/packages/my-jetpack/src/class-products.php#L95).
I think keeping the `absence` word there does make sense, even when defining the styles. Because, otherwise, the `absence` product status could be interpreted as the product is absent, confusing the meaning. The product is there, but what's absent is the plugin.
No strong opinion, though. We can change the style name if you prefer.
Sorry I saw the comment before to merge.

Reply:
Np! Let's see there in the upcoming work